README.md

```diff
@@ -1,11 +1,8 @@
 # ttl.sh
 
-An ephemeral container registry for CI workflows.
+## An ephemeral container registry for CI workflows.
```

Comment:
should this be a heading considering it has nothing under it?

Reply:
I thought it looked a little better, but I can change it. the entire readme needs revamping tbqh

Reply:
i'm fine either way! no need to change it if it looks better 👍